model-server/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM registry.access.redhat.com/ubi8/openjdk-11:1.18-2.1705602259
 WORKDIR /usr/modelix-model
 EXPOSE 28101
+HEALTHCHECK CMD curl --fail http://localhost:28101 || exit 1
```

Comment:
We have a dedicated health endpoint that can be used here. Cf. https://github.com/modelix/modelix.core/blob/aa16a542008905c49cc465a535a2557aead9369d/docs/global/modules/core/pages/howto/usage-model-server.adoc?plain=1#L70

```suggestion
HEALTHCHECK CMD curl --fail http://localhost:28101/health
```

What was the reason to add `|| exit 1`? Curl would fails with a non-zero exit code in any case.

Reply:
Oops, now the remaining comments pointing to that loaded :see_no_evil:

Reply:
> What was the reason to add || exit 1? Curl would fails with a non-zero exit code in any case.

The reason is that curl may exit with any code https://timi.eu/docs/anatella/5_1_9_1_list_of_curl_error_codes.html but the health check should exit with either 0, 1 or 2. But it should not even be exited with 2 :D 


![image](https://github.com/modelix/modelix.core/assets/36853386/86388c06-13d8-42b2-ab93-94d5ddad5d12)

Reply:
Ah right, didn't know that Docker healthchecks are that picky.